helpers.js

```diff
@@ -492,6 +492,16 @@ const getArmletClient = (ethAddress, password, clientToolName = 'truffle') => {
  * @param {Object} config - truffle configuration object.
  */
 async function analyze(config) {
+
+    try {
+      const projectConfig = require([config.working_directory,
+				       'truffle-security.json'].join ('/'));
+      config = Object.assign(config, projectConfig);
+    } catch (ex) {
+      // TODO how chatty should this logging be?
+      // console.log("No truffle security configuration file found");
```

Comment:
Logging this seems appropriate if `config.debug > 1`.

Reply:
thanks for this!